stores/moderationSubmissionsStore.ts

```diff
@@ -17,6 +17,11 @@ export const useModerationSubmissionsStore = defineStore(
         const selectedSubmissionId: Ref<string> = ref('')
         const selectedSubmissionData: Ref<Submission | undefined> = ref()
         const filteredSubmissionDataForListComponent: Ref<Submission[]> = ref([])
+        const didMutationFail: Ref<boolean> = ref(false)
+
+        function handleDidMutationFail(newValue: boolean) {
```

Comment:
`handle` is a standard convention for a function handling events. Since this is just setting the value, let's rename it
```suggestion
        function setDidMutationFail(newValue: boolean) {
```